### DISCUSSION TITLE
AI與自駕車

### DISCUSSION THREAD

**Discussion by 109302003** [1101-GS4719-C/Python-Community-GS4719-C#96] [General]
時下最熱門的自駕車，利用ai 機器學習，配備多種感測器偵測即時資訊。現在特斯拉的全自動輔助駕駛，可以偵測道路狀況，協助矯正偏離車道、倒車，控制方向盤、煞車和油門等駕駛功能，但無法處理道路上所有狀況，我覺得在台灣路況差，道路錯綜複雜，要普及自駕車非常困難。

[https://www.eettaiwan.com/20201012ta31-role-of-ai-in-autonomous-vehicles/]()

**Comment by Jie2222** [comment]
我覺得**全自動化**目前不太可能，但是**半自動輔助**感覺可以，像是高速公路就滿適合，車況比較單純

**Comment by vvvioletta** [comment]
我覺得在臺灣的平面道路要實現level 5還有一個難處，路上太多雜訊會干擾AI判讀道路狀況，像是塗鴉、亂貼的傳單、售屋的大立牌等等
